### DISCUSSION TITLE
AH2 skin is not working on Kodi 21

### DISCUSSION THREAD

**Discussion by Lagschman** [jurialmunkey/skin.arctic.horizon.2#1025] [Discussion]
I updated to KODi 21 and since then I always get an error message that the skin is not compatible with KODi 21. What can I do ?

**Comment by SeeThisIsMe** [comment]
I have the same. Using libreelec v12 and Kodi omega v21 I get errors and can't use it. The skin is listed as having an update available but it fails to install due to incompatibility errors.

**Reply by sgeg06** [reply]
Hi errors can be fixed by update dependencies from dependency management not the repository directly. Hope that will help you
